01-node-tutorial/answers/11-fs-async.js

```diff
@@ -0,0 +1,26 @@
+const { readFile, writeFile } = require('fs')
+
+readFile('../content/first.txt', 'utf8', (err, result) => {
+    if (err) {
+        console.log(err);
+        return
+    }
+    // console.log(result);
+    const first = result;
+    readFile('../content/second.txt', 'utf8', (err, result) => {
```

Comment:
Good work! You understood the way callbacks nest. However, these are readFiles, but you need to do writeFile 3 times in a similar way, and including the 'a' flag for appending

Reply:
changes done